docs/enterprise/thales-install-guide.md

```diff
@@ -0,0 +1,61 @@
+# Using the Thales nShield Connect Hardware Security Module with Chain Core
```

Comment:
Perhaps we should reduce the number of words in the h1. All of our other doc pages use a single line for the h1.